web-common/src/components/typography/RadixH1.svelte

```diff
@@ -0,0 +1,3 @@
+<h1 class="font-extrabold text-5xl tracking-tight">
```

Comment:
Yes, because our Figma design system currently has two typography categories – a Rill category and a Radix category. We discussed with Janet that a better approach would be to have only 1 category, the Rill category, which then may use Radix components in implementation.

Hopefully we can get to that state, I'd delete all the `Radix[Element].svelte` components, and we'd be left exclusively with `[Element].svelte` components.